### ISSUE TITLE
Name not available in the contributors list..

### ISSUE THREAD

**Issue by rakesh-eltropy** [markbates/goth#408]
@markbates  I contributed a lot in the project in 2016. Today, I was randomly checking in the contributor list to find out that I was not there. I had introduced refresh token and added around 10 different providers (Amazon,Box,Onedrive,Salesforce and Yammer providers etc). 

Could you please throw some light on this.

**Comment by bentranter** [comment]
Hey @rakesh-eltropy, sorry about this – if you would like to open a PR to add yourself to the list, I'd be happy to merge it. If not, I can add you sometime later this week.

**Comment by techknowlogick** [comment]
Yeah, that's rather odd. You are marked as a collaborator on this project, however no commits are associated to your user: https://github.com/markbates/goth/commits?author=rakesh-eltropy
